templates/projects/index.html

```diff
@@ -139,8 +139,8 @@ <h2 class="p-muted-heading">Some projects we care about</h2>
 
 <section class="p-strip">
   <div class="row">
-    <div class="col-6">
-      <div class="row">
+    <div class="col-6 p-divider__block">
+      <div class="row p-divider">
```

Comment:
Don't think you need this `p-divider` class.

Reply:
good spot - amended